src/compile/references.js

```diff
@@ -217,27 +217,11 @@ module.exports = function(Velocity, utils) {
 
       var id = property.id;
       var ret = '';
-
-      // getter 处理
-      if (id.indexOf('get') === 0 && !(id in baseRef)) {
-        if (id.length === 3) {
-          // get('address')
-          ret = getter(baseRef, this.getLiteral(property.args[0]));
-        } else {
-          // getAddress()
-          ret = getter(baseRef, id.slice(3));
```

Comment:
Very sorry to reply about this so later. I think this code should be retained. Maybe you should try to run `npm test`.